swarm/pss/testdata/addpsstodiscoverytestsnapshot.pl

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/perl
+
+use JSON;
+
+my $f;
+my $jsontext;
+my $nodelist;
+my $network;
+
+open($f, "<", $ARGV[0]) || die "cant open " . $ARGV[0];
+while (<$f>) {
+	$jsontext .= $_;
+}
+close($f);
+
+$network = decode_json($jsontext);
+$nodelist = $network->{'nodes'};
+
+for ($i = 0; $i < 0+@$nodelist; $i++) {
+	my $svc = $$nodelist[$i]{'node'}{'config'}{'services'};
+	pop(@$svc);
+	push(@$svc, "pss");
+	push(@$svc, "bzz");
+}
+
+print encode_json($network);
```

Comment:
I'd recommend doing this with [`jq`](https://stedolan.github.io/jq/) rather than perl:

```
jq '.nodes |= map(.node.config.services = ["bzz", "pss"])' snapshot.json
```

Reply:
And to be honest, we should probably update `net.Load` so that it supports overriding the services at load time.

Reply:
yes or just leave it to the tests

Reply:
Actually even more architecture neutral would be:

`sed -e 's/\(\"services\"\):\["discovery\"]/\1:["pss","bzz"]/'`

(my arch distro, for example, didn't include `jq` by default)